README.md

```diff
@@ -1 +1,132 @@
-# school-library
\ No newline at end of file
+## School-libray-Ruby
+
+<a name="readme-top"></a>
+<div align="center">
+    <img src="" alt="logo" width="700"  height="auto" />
```

Comment:
- [ Optional] Well done implementing all the classes, I recommend deleting the logo or replace it with your own so that your Readme will look more professional.

Reply:
Thanks for your recommendation!